config/default.ts

```diff
@@ -1,7 +1,11 @@
-export default ({ 
-    APIGenerated: {
-      port: process.env.PORT || 3000,
-      enforceWhitelist: process.env.WHITELIST || "FALSE",
-      mainnet: process.env.MAINNET || "FALSE",
-    }
-  } as ConfigType);
\ No newline at end of file
+export default {
+  contract: {
```

Comment:
In source and executor analogous config key is named `m1`. II think it's not a bad idea to keep it for consistency. However, `sidechain` may be the best key name.

But certainly change the current name please. Nodeurl or chainid are not properties of contract as this implies.